### PR TITLE
Gui: Add can hide gui after boot game.

### PR DESCRIPTION
**Pull request by Zangetsu38**
detected with test rtc, gui hide all time up part on screen for game, i have so thinked can add possibility for can hide gui in boot game directly without press G

### CODE REVIEW

File: src/emulator/gui/src/main_menubar.cpp [hunk 1]
```diff
@@ -52,6 +52,15 @@ static void draw_config_menu(HostState &host) {
             }
             ImGui::EndMenu();
         }
+        if (ImGui::BeginMenu("Gui State")) {
+            if (ImGui::MenuItem("Hide Gui", nullptr, !host.cfg.gui_state)) {
```
Comment (Rinnegatamante) [inline src/emulator/gui/src/main_menubar.cpp]: Use a checkbox instead of two different voices.

Comment (Zangetsu38) [reply src/emulator/gui/src/main_menubar.cpp]: ah yes

Comment (Zangetsu38) [reply src/emulator/gui/src/main_menubar.cpp]: Done :)

File: src/emulator/host/src/app.cpp [hunk 2]
```diff
@@ -305,6 +305,14 @@ ExitCode load_app(Ptr<const void> &entry_point, HostState &host, const std::wstr
         return ModuleLoadFailed;
     }
 
+    // Gui State
+    if (!host.cfg.gui_state) {
+        auto &display = host.display;
```
Comment (VelocityRa) [inline src/emulator/host/src/app.cpp]: may as well do
```suggestion
        auto &imgui_render = host.display.imgui_render;
```

File: src/emulator/host/src/config.cpp [hunk 3]
```diff
@@ -126,6 +127,7 @@ ExitCode deserialize(Config &cfg) {
     get_yaml_value(config_node, "log-active-shaders", &cfg.log_active_shaders, false);
     get_yaml_value(config_node, "log-uniforms", &cfg.log_uniforms, false);
     get_yaml_value(config_node, "pstv-mode", &cfg.pstv_mode, false);
+    get_yaml_value(config_node, "show-gui", &cfg.show_gui, false);
```
Comment (VelocityRa) [inline src/emulator/host/src/config.cpp]:
```suggestion
    get_yaml_value(config_node, "show-gui", &cfg.show_gui, true);
```

File: src/emulator/host/include/host/config.h [hunk 4]
```diff
@@ -40,6 +40,7 @@ struct Config {
     bool log_uniforms = false;
     std::vector<std::string> lle_modules;
     bool pstv_mode = false;
+    bool show_gui = false;
```
Comment (VelocityRa) [inline src/emulator/host/include/host/config.h]:
```suggestion
    bool show_gui = true;
```

File: src/emulator/host/src/config.cpp (hunk 3 above)
Comment (Zangetsu38) [reply src/emulator/host/src/config.cpp]: ah for that no, i want leave it on false by default, for all user is better hide gui in defaut setting

File: src/emulator/host/include/host/config.h (hunk 4 above)
Comment (Zangetsu38) [reply src/emulator/host/include/host/config.h]: same for here, like i have said, gui hide up part game, so hide it by defaut is normal

File: src/emulator/host/src/config.cpp (hunk 3 above)
Comment (VelocityRa) [reply src/emulator/host/src/config.cpp]: Aight I guess it's fine if we can still enable it from the settings.

**Pull request merged**